k8s/src/main/scala/io/buoyant/k8s/v1.scala

```diff
@@ -96,7 +96,7 @@ package object v1 {
       HighResTimer.Default,
       stats,
       RetryBudget.Infinite
-    )
+    ) andThen new LogFailureFilter(log, stats)
```

Comment:
totally minor tioli: i'd be inclined to separate this from infiniteRetryFilter and have i.e. `val logFailureFilter = ...` and then layer it in below as `retry and then logFailureFilter andThen client`

(Errors should still be logged, etc when retries aren't enabled).